docs/README.md

```diff
@@ -215,356 +153,385 @@ for these executables in your `$PATH`.
+`forc index new`
+    Create a new index.
+`forc index init`
+    Create a new index in an existing directory.
+`forc index start`
+    Start a local indexer service.
+`forc index build`
+    Build your index.
+`forc index deploy`
+    Deploy your index.
+`forc index remove`
+    Stop a running index.
+```
 
-`cd contracts/ && forc new greeting`
+> IMPORTANT: If you want more details on how this index works, checkout our [block explorer index example](https://fuellabs.github.io/fuel-indexer/master/examples/block-explorer.html).
 
-Write a "greeting" type of Sway smart contract.
+#### 2.3 Deploying our index
 
-```sway
-// src/main.sw
-contract;
+By now we have a brand new index that will index some blocks and transactions, but now we need to build and deploy it in order to see it in action.
 
-use std::logging::log;
+##### 2.3.1 Starting an indexer service
 
-struct Person {
-    name: str[32],
-}
+- To start an indexer service, we'll be spinning up Postgres and Fuel indexer containers via docker compose. Our indexer service will connect to Fuel's `beta-2` network so that we can index blocks and transactions from an _actual_ Fuel node. We'll use the `docker compose` file below, and spinning everything up with `docker compose up`.
 
-struct Greeting {
-    id: u64,
-    greeting: str[32],
-    person: Person,
-}
+> IMPORTANT: Ensure that any local Postgres instance that is running on port `5432` is stopped.
+>
+> You can open up a `docker-compose.yaml` file in the same directory as your index manifest, and paste the YAML content below to this `docker-compose.yaml` file.
 
-abi Greet {
-    fn new_greeting(id: u64, greeting: str[32], person_name: str[32]);
-}
+```text
+version: "3.9"
+services:
+  postgres:
+    image: postgres:latest
+    ports:
+      - "5432:5432"
+    volumes:
+      - .:/usr/local/postgres
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - PGUSER=postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-U", "postgres", "-d", "postgres"]
+      interval: 30s
+      timeout: 60s
+      retries: 5
+      start_period: 80s
+  fuel-indexer:
+    image: ghcr.io/fuellabs/fuel-indexer:latest
+    command: bash -c "sleep 2 && ./fuel-indexer --fuel-node-host node-beta-2.fuel.network --fuel-node-port 80 --postgres-host postgres --postgres-password postgres --graphql-api-host 0.0.0.0"
+    ports:
+      - "29987:29987"
+    volumes:
+      - .:/usr/local/fuel-indexer
+    depends_on:
+      - postgres
+```
+
+##### 2.3.2 Deploying your index to your Fuel indexer service
+
+With our database and Fuel indexer indexer containers up and running, we'll deploy the index that we previously created. If all goes well, you should see the following:
 
-impl Greet for Contract {
-    fn new_greeting(id: u64, greeting: str[32], person_name: str[32]) {
-        log(Greeting{ id, greeting, person: Person{ name: person_name }});
+```text
+forc-index deploy --manifest hello_index.manifest.yaml --url http://0.0.0.0:29987
```

Comment:
Similar comment here about deploying `hello_index` but mentioning the block explorer above and then querying for the block explorer types in the next section.

Reply:
Same response as  [here](https://github.com/FuelLabs/fuel-indexer/pull/491#discussion_r1084091607)